acquisitions/src/main/resources/thunderjet/mod-consortia/features/tenant.feature

```diff
@@ -24,79 +35,89 @@ Feature: Tenant object in mod-consortia api tests
 
     # attempt to create a tenant without a code
     Given path 'consortia', consortiumId, 'tenants'
+    And param adminUserId = centralAdmin.id
     And request { id: '1234', name: 'test', isCentral: false }
     When method POST
     Then status 422
     And match response == { errors: [{ message: "'code' validation failed. must not be null", type: '-1', code: 'tenantValidationError'}] }
 
     # attempt to create a tenant without a name
     Given path 'consortia', consortiumId, 'tenants'
+    And param adminUserId = centralAdmin.id
     And request { id: '1234', code: 'ABC', isCentral: false }
     When method POST
     Then status 422
     And match response == { errors: [{ message: "'name' validation failed. must not be null", type: '-1', code: 'tenantValidationError'}] }
 
     # attempt to create a tenant without isCentral
     Given path 'consortia', consortiumId, 'tenants'
+    And param adminUserId = centralAdmin.id
     And request { id: '1234', code: 'ABC', name: 'test' }
     When method POST
     Then status 422
     And match response == { errors: [{ message: "'isCentral' validation failed. must not be null", type: '-1', code: 'tenantValidationError'}] }
 
 #    # attempt to create a tenant with a name that has length more than 150 characters and a code with more than 3 characters
 #    Given path 'consortia', consortiumId, 'tenants'
-#    And request { id: '12345', code: 'TTTT', name: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec auctor, nisl eget ultricies lacinia, nisl nisl aliquet nisl, eget aliquet nisl nisl eget nisl. Donec auctor, nisl eget ultricies lacinia, nisl nisl aliquet nisl, eget aliquet nisl nisl eget nisl.', isCentral: true }
+#    And param adminUserId = centralAdmin.id
+#    And request
+#    """
+#    {
+#      id: '12345',
+#      code: 'TTTT',
+#      name: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec auctor, nisl eget ultricies lacinia, nisl nisl aliquet nisl, eget aliquet nisl nisl eget nisl. Donec auctor, nisl eget ultricies lacinia, nisl nisl aliquet nisl, eget aliquet nisl nisl eget nisl.',
+#      isCentral: true
+#    }
+#    """
 #    When method POST
 #    Then status 422
 #    And match response.errors[*].message contains ['\'code\' validation failed. Invalid Code length: Must be of 3 alphanumeric characters', '\'name\' validation failed. Invalid Name: Must be of 2 - 150 characters']
 
-  Scenario: Do Create, Read, Update for 'central' tenant (isCentral = true)
-    # create 'central' tenant
+  @Positive
+  Scenario: Do POST a tenant, GET list of tenant(s) (isCentral = true)
+    # get tenants of the consortium (before posting any tenant)
+    Given path 'consortia', consortiumId, 'tenants'
+    When method GET
+    Then status 200
+    And match response == { tenants: [], totalRecords: 0 }
+
+    # post a tenant with isCentral=true ('central' tenant)
     Given path 'consortia', consortiumId, 'tenants'
+    And param adminUserId = centralAdmin.id
```

Comment:
for adding central tenant it is not required to provide adminUserId param

Reply:
Basically now we have validation that requires this param in for central and non-central tenants, @azizbekxm fixed it in scope of PR https://github.com/folio-org/mod-consortia/pull/50. After merging this PR we can delete this param for central tenant